frontend/cypress/e2e/hwcr-outcome-equipment.cy.ts

```diff
@@ -39,7 +39,6 @@ describe("HWCR Outcome Equipment", () => {
           "#equipment-address-div",
           "#equipment-coordinates",
           "#equipment-coordinates",
-          "#equipment-officer-set-div",
```

Comment:
This is failing and I'm currently investigating why. Removing it is a test to see if I can narrow down the issue.

Reply:
@nayr974  - there does still seem to be a bug here around the validation... 

Steps to reproduce:
- Navigate to a complaint and add equipment (without a removed date)
- Navigate to the list and bring up the QC Modal on the complaint
- Validation message appears (good!)
- Close the QC Modal and open up any other QC Modal... they all display the message

Variation:
- Repeat the setup above...
- Hard refresh the application
- Launch the QC Modal on the complaint with the equipment in the field
- Validation Message does not appear

Reply:
Looking better... I was able to get it into one silly situation but this seems like such an edge case it might not be worth fixing:
- Navigate to open complaint with no outcome data
- Open a section (e.g. Notes)
- Click back to the list
- Attempt to Quick Close the list

At this point it tells to complete the section that I started working on and abandoned... going into the quick close modal again from the details page works as expected.

Reply:
@afwilcox Each section will now set it's edit state to false when its component unmounts :)